pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -260,6 +296,29 @@
 				</buttonbar>
 			</panel>
 
+			<tab><translate>Experimental</translate></tab>
+			<panel>
+				<h2><translate>Material system</translate></h2>
+				<row>
+					<h3><translate>Material system</translate></h3>
+					<input cvar="r_materialSystem" type="checkbox" />
+					<p><translate>Use a new system that significantly improves performance. Some drivers may fail with this on (e. g. Mesa radeonsi and AMD proprietary drivers).</translate></p>
```

Comment:
I would append some `&nbsp;<translate>Experimental.</translate>` string on all those options instead.

Reply:
I added it to the first one, adding to all seems redundant though?

Reply:
I prefer for everything experimental be marked as is.

Reply:
Sure, I updated it now.